play/PlayerRules/CharacterCreation/Class-Specific Documentation/Power Suits/Chassis.txt

```diff
@@ -0,0 +1,19 @@
+		Light Chassis
```

Comment:
I think we should consider throwing the distinctions out of the window. Then we can name things like "ECSC Jaeger Assault Chassis". I'm sure we'll still use terms like "light, medium, heavy" for descriptors. Also opens the door wide open to unique sweet loot